UnitystationLauncher/GameCommunicationPipe/PipeHubBuildCommunication.cs

```diff
@@ -13,18 +13,31 @@ namespace UnitystationLauncher.GameCommunicationPipe;
 
 public class PipeHubBuildCommunication : IDisposable
 {
-    private NamedPipeServerStream _serverPipe;
-    private StreamReader? _reader;
-    private StreamWriter? _writer;
+    private static NamedPipeServerStream? _serverPipe;
+    private static StreamReader? _reader;
+    private static StreamWriter? _writer;
 
     public PipeHubBuildCommunication()
     {
+        _serverPipe?.Close();
+        _serverPipe?.Dispose();
+        _reader?.Close();
+        _reader?.Dispose();
+        _writer?.Close();
+        _writer?.Dispose();
+
         _serverPipe = new("Unitystation_Hub_Build_Communication", PipeDirection.InOut, 1,
             PipeTransmissionMode.Byte, PipeOptions.Asynchronous);
     }
 
-    public async Task StartServerPipe()
+    public static async Task StartServerPipe()
     {
+        if (_serverPipe == null)
+        {
+            _serverPipe = new("Unitystation_Hub_Build_Communication", PipeDirection.InOut, 1,
```

Comment:
turn this string into a constant

Reply:
It's only used twice tho?

Reply:
twice is a lot where mistakes can happen